data_validation/cli_tools.py

```diff
@@ -291,6 +291,12 @@ def _configure_run_parser(subparsers):
         "-filters",
         help="Filters in the format source_filter:target_filter",
     )
+    run_parser.add_argument(
```

Comment:
We should add documentation on this CLI option in the README and mention that it defaults to 'table'

Reply:
Done.